src/Advection/vector_invariant_advection.jl

```diff
@@ -175,10 +175,19 @@ Base.show(io::IO, a::VectorInvariant{N, FT}) where {N, FT} =
 nothing_to_default(user_value; default) = isnothing(user_value) ? default : user_value
 
 """
-    WENOVectorInvariant(; upwinding = nothing,
-                          multi_dimensional_stencil = false,
-                          weno_kw...)
-
+    WENOVectorInvariant(FT = Float64; 
+                        upwinding = nothing,
+                        vorticity_stencil = VelocityStencil(),
+                        order = nothing,
+                        vorticity_order = nothing,
+                        vertical_order = nothing,
+                        divergence_order = nothing,
+                        kinetic_energy_gradient_order = nothing, 
+                        multi_dimensional_stencil = false,
+                        weno_kw...)
+
+Return a vector-invariant weighted essentially non-oscillatory (WENO) scheme.
+See [`VectorInvariant`](@ref) and [`WENO`](@ref) for kwargs definitions.
```

Comment:
@simone-silvestri what does `multi_dimensional_stencil` do?

Reply:
When I asked about it in https://github.com/CliMA/Oceananigans.jl/discussions/3856 he said that
>  multi-dimensional stencils implement a 2D horizontal stencil instead of a 1D stencil that performs a centered 5th-order WENO reconstruction of vorticity, divergence and kinetic energy in the tangential direction. In theory, it is more accurate, but it is very computationally expensive and doesn't seem to improve the results much, so I haven't found a good use for this reconstruction.